README.md

```diff
@@ -1 +1,44 @@
-# siddhi-io-nats
\ No newline at end of file
+﻿siddhi-io-nats
+======================================
+
```

Comment:
Add a basic description. Do a mkdocs build and serve optionally to check all doc are generated correctly

Reply:
Resolved with the commit 9c2f344de3cc07e51588e9804db3c8a37d8ae24b.

Reply:
Features tag should be added by default in each build. See why that is not added